css/about.scss

```diff
@@ -74,6 +74,20 @@ h1, h2, h3, h4 {
   }
 }
 
+#freshers-banner {
```

Comment:
Could we make the bar a bit thicker vertically? The text looks a little too close to the edge of the bar to me

Reply:
does latest commit resolve?

Reply:
Yup!